src/pages/user-profile-view/user-profile-view.scss

```diff
@@ -30,4 +30,52 @@ user-profile-view {
       @include header();
     }
   }
+
+  .delete-icon {
+    text-align: center;
+    color: $almost-black;
+    margin: auto;
+    cursor: pointer;
+    
+    &:hover {
+      color: $error-color
+    }
+
+    &:active {
+      color: $medium-grey
+    }
+  }
+
+  .centered {
+    text-align: center;
+  }
+}
+
+div.modal-button-group {
```

Comment:
Is there a reason this is at the top level? Should we move it somewhere more common if it doesn't need to be namespaced under a component?